pkg/scanners/clairify/clairify.go

```diff
@@ -402,9 +402,14 @@ func (c *clairify) addScan(image *storage.Image, uncertifiedRHEL bool) error {
 
 // GetVulnerabilities retrieves the vulnerabilities present in the given image
 // represented by the given components and scan notes.
-func (c *clairify) GetVulnerabilities(image *storage.Image, components *clairGRPCV1.Components, notes []clairGRPCV1.Note) (*storage.ImageScan, error) {
+func (c *clairify) GetVulnerabilities(image *storage.Image, components any, notes []clairGRPCV1.Note) (*storage.ImageScan, error) {
+	clairComponents, ok := components.(*clairGRPCV1.Components)
```

Comment:
the fact it's called `clair` when it's not clair is unfortunate. I was going to ask to change `clairComponents` but the type makes it sound like `clair`